### ISSUE TITLE
Understanding top_rate, other_rate wrt goss

### ISSUE THREAD

**Issue by nickwan** [Microsoft/LightGBM#1477]
Was trying to find more documentation on how top_rate and other_rate are used in the goss boosting parameter but didn't seem to find much. How is top_rate and other_rate different from learning_rate? 

Tried searching through closed topics but couldn't find any mentions. Perhaps not the best spot for this question (not really an "issue") but unsure where else to ask.

**Comment by guolinke** [comment]
refer to paper : http://papers.nips.cc/paper/6907-lightgbm-a-highly-efficient-gradient-boosting-decision-tree.pdf
